xinference/model/llm/core.py

```diff
@@ -216,97 +306,240 @@ def __init__(
         self._user_name: str = user_name
         self._assistant_name: str = assistant_name
 
-    def _to_prompt(
+    def chat(
```

Comment:
Can you move llamacpp and pytorch models into 2 different files respectively?

Reply:
done